<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLinkWindow</class>
 <widget class="QWidget" name="AddLinkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="inpType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Pictures</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Book</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Document</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Audio</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Video</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Project</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Website</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of object:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="btnWebCheck">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Web Link</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpLink">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblLink">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Link:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseFile">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblComment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inpComment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
